<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>2000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="LoadButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>530</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GenerateButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>560</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенерировать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SolveButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>590</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Решить</string>
    </property>
   </widget>
   <widget class="QLabel" name="startPointLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>530</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Начальная точка</string>
    </property>
   </widget>
   <widget class="QLabel" name="endPointLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>560</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Конечная точка</string>
    </property>
   </widget>
   <widget class="QLabel" name="startXlabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>530</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="startYlabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>530</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLabel" name="rowsLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>590</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Строки</string>
    </property>
   </widget>
   <widget class="QLabel" name="columnsLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>590</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Столбцы</string>
    </property>
   </widget>
   <widget class="QLabel" name="endXlabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>560</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="endYlabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>560</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="startX">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>530</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="startY">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>530</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="endX">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>560</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="endY">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>560</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="rows">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>590</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="columns">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>590</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="MazeWidget" name="maze_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MazeWidget</class>
   <extends>QWidget</extends>
   <header location="global">mazewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
